### DISCUSSION TITLE
Choosing a local image for workspace icon

### DISCUSSION THREAD

**Discussion by Heinrich-XIAO** [zen-browser/desktop#4063] [Ideas]
I feel like this would be a good idea.

**Comment by 12th-devs** [comment]
You can add other emojis besides the ones given if you go into about:config and search workspaces or something, I don't remember off the top of my head. Then you can edit the list given, just make sure you keep the formatting the same.

**Reply by Heinrich-XIAO** [reply]
Yeah, but they don't let you make your own icon unless you make a new font.

**Comment by FlakyBlueJay** [comment]
+1, I have workspaces dedicated to a singular site, and I have to choose an emoji for it that doesn't exactly fit.

Sidebery and Simple Tab Groups both allow this, FWIW.
